TraitsExpanded/TraitSets/BigCharacterTraitSet.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.Core;
+using TaleWorlds.MountAndBlade.Diamond;
+using TraitsExpanded.Traits;
+
+namespace TraitsExpanded.TraitSets
+{
+    public class BigCharacterTraitSet : TraitSet
+    {
+        public override string ID { get; set; } = "BigCharacterTraitSet";
+
+        public BigCharacterTraitSet(CharacterObject characterObject)
+        {
+            Init(characterObject);
+        }
+        
+        public override void Init(CharacterObject character)
```

Comment:
why not just do these things in the constructor itself?

Reply:
I wanted to keep a proper function defining what is happening.

Reply:
how do you mean? a constructor is the initialization of an object, so just setting members and stuff like that typically belongs there. do you anticipate other kinds of initialization will be required - stuff that isn't appropriate to do during a constructor (async io, etc)?